packages/react-app/src/components/giveaways/participation/ParticipantEntry.tsx

```diff
@@ -13,6 +15,7 @@ const ParticipantEntry = ({
   actionAllowed,
   onUpdateState,
 }: ParticipantEntryProps) => {
+  const [isLoading, setIsLoading] = useState<string>('no action');
```

Comment:
Can we have a blank line after this one please

Reply:
Also, can we call this `loadingStatus` or something similar, since this is not a boolean